src/actions/gameAction.js

```diff
@@ -0,0 +1,17 @@
+export const SCORE = 'SCORE';
+export const LOGIN = 'LOGIN';
+
+const gameAction = (point) => ({
+  type: SCORE,
+  score: point,
```

Comment:
Não sei se faz sentido, ou se é uma boa prática, mas o que acha de deixar a informação do score dentro de uma chave de payload igual foi feito na loginAction?
```suggestion
  payload: {
    score: point,
  }
```